src/ast/mod.rs

```diff
@@ -4828,19 +4861,25 @@ impl fmt::Display for Function {
             };
             write!(
                 f,
-                "{}({}{}{order_by}{})",
+                "{}({}{}{order_by}{}{})",
                 self.name,
                 if self.distinct { "DISTINCT " } else { "" },
                 display_comma_separated(&self.args),
                 display_comma_separated(&self.order_by),
+                match self.null_treatment {
+                    Some(NullTreatmentType::FunctionArg(null_treatment)) => {
+                        format!(" {null_treatment}")
+                    }
+                    _ => "".to_string(),
+                }
```

Comment:
I think you can avoid this seeming allocation in `"".to_string()` by using [`Cow`](https://doc.rust-lang.org/std/borrow/enum.Cow.html):

```suggestion
                match self.null_treatment {
                    Some(NullTreatmentType::FunctionArg(null_treatment)) => {
                        Cow::Owned(format!(" {null_treatment}"))
                    }
                    _ => Cow::Borrowed("")
                }
```

Reply:
Done!